src/mono/mono/utils/mono-proclib.h

```diff
@@ -19,5 +19,12 @@ MONO_API
 int
 mono_cpu_count (void);
 
+MONO_API 
+int
+mono_cpu_limit (void);
```

Comment:
does this need to be `MONO_API`?
```suggestion
int
mono_cpu_limit (void);
```

Reply:
pretty sure it does but @nealef would you mind checking?

Reply:
It is used in place of `mono_cpu_count()` in a couple of spots so I think it needs to have the same attributes.